wiki/Article_Style_Guide/en.md

```diff
@@ -499,29 +568,30 @@ Use of this type of link may include:
 
 ### Link Text
 
-Do not use the word `here` as the link text.
-As it may seem convenient to use, it could be misleading.
-Instead, take the page's title (from the top of page, or from the `<title>` tag) and use that as the link text.
+Do **not** use the word `here` as the link text.
+As it may seem convenient to use, this may cause confusion and/or could be misleading.
+
+Instead, take the page's title (from the top of page or from the `<title/>` tag) and use that as the link text.
 
 ### Section Links
 
-All section links follow these rules:
+All section links follow these strict rules:
 
 - all letters **must** use lowercase letters
 - all special characters (e.g. `!`, `?`, `,`, `"`, `'`) **must** be replaced with a hyphen (`-`)
   - if one of the spacial characters is the last character in the section name, remove it, but do not add a hyphen (`-`).
```

Comment:
would it be worth the effort to have them link wrong in GitHub but not in osu-web?